src/bundle/Resources/config/services/services.yaml

```diff
@@ -0,0 +1,10 @@
+services:
```

Comment:
I would create a dedicated file for this or move it to `src/bundle/Resources/config/services.yaml`